yaml_files/simphony_metadata.yml

```diff
@@ -436,6 +484,50 @@ CUDS_KEYS:
     definition: Periodic boundary condition (PBC)
     models: [CUBA.ELECTRONIC, CUBA.ATOMISTIC, CUBA.MESOSCOPIC, CUBA.CONTINUUM]
 
+  SHEAR_STRESS_POWER_LAW_SLIP_VELOCITY:
+    parent: CUBA.CONDITION
+    definition: Shear stress power law dependant slip velocity boundary condition
+    models: [CUBA.CONTINUUM]
+    CUBA.VARIABLE:
```

Comment:
I am a bit baffled by the generic nature of this entity... Maybe I don't get its point.